torch/_torch_docs.py

```diff
@@ -4431,6 +4431,27 @@ def merge_dicts(*dicts):
 """)
 
 
+add_docstr(torch.result_type,
+           r"""
+result_type(tensor, other) -> dtype
+
+Returns the :class:`torch.dtype` that would result from performing an arithmetic
+operation on the provided input tensors. See :class:`torch.dtype` documentation
```

Comment:
can we link directly to that specific section?

Reply:
added a reference in the doc and a direct link to it